src/comp_loinc/loinc_builder_steps.py

```diff
@@ -790,7 +789,10 @@ def save_to_owl(
         if owl_file_path is None:
             owl_file_path = Path(self.runtime.current_module.name + ".owl")
         if not owl_file_path.is_absolute():
-            owl_file_path = self.configuration.output / owl_file_path
+            outdir = self.configuration.output
+            if self.configuration.fast_run:
+                outdir = outdir / "fast_run"
```

Comment:
## General review
I ran the full `--fast-run` build and it ran without error, and I found files in the directory expected.